mixpanelUtilities.go

```diff
@@ -148,7 +148,7 @@ func (m *MixpanelClient) TrackEvent(ctx context.Context, eventName string, prope
 	event := m.Client.NewEvent(eventName, m.DistinctID, finalProperties)
 	err := m.Client.Track(ctx, []*mixpanel.Event{event})
 	if err != nil {
-		log.Printf("Mixpanel error; err=%v", err)
+		log.Printf("mixpanel error; err=%v", err)
 		m.Enabled = false
 	}
 	return err
```

Comment:
Are go files allowed to contain a capital letter? Like mixpanel**U**tilities.go?

Reply:
You're right best practice is all lowercase , or underscore separation, I'll fix that as well